lib/APPSensorFusion/src/ErrorState.h

```diff
@@ -45,6 +45,7 @@
  *****************************************************************************/
 #include <stddef.h>
 #include <IState.h>
+#include "StartupState.h"
```

Comment:
Maybe move this to cpp?

Reply:
I got an error when trying to build it when the include is inside the cpp:
`error: 'M_PI' was not declared in this scope`